client/src/app/shared/models/poll/poll-constants.ts

```diff
@@ -1,5 +1,39 @@
 import { marker as _ } from '@biesbjerg/ngx-translate-extract-marker';
 
+/**
+ * The main interface to describe everything related to polls
+ * This is a unification of the data received in the projector and ViewPoll; both
+ * use this interface, so the poll services can work on this shared view on the data.
+ */
+export interface PollData {
+    pollmethod: PollMethod;
+    state: PollState;
+    onehundred_percent_base: PollPercentBase;
+    votesvalid: number;
+    votesinvalid: number;
+    votescast: number;
+    type: PollType;
+    entitled_users_at_stop: EntitledUsersEntry[];
+    options: OptionData[];
+    global_option: OptionData;
+    getContentObjectTitle: () => string | null;
+}
+
+export interface OptionTitle {
+    title: string;
+    subtitle?: string;
+}
+
+export interface OptionData {
+    // content_object?: {};
+    getOptionTitle: () => OptionTitle;
+    // text?: string;
+    yes?: number;
+    no?: number;
+    abstain?: number;
+    weight?: number;
+}
```

Comment:
I know, there are some other interfaces in this file, but I think, that poll-**constants**.ts is not the right place for interfaces. Maybe you could put these interfaces into an own file.